admin-dev/themes/default/template/controllers/translations/helpers/view/main.tpl

```diff
@@ -197,7 +197,7 @@
 			</div>
 		</div>
 	</form>
-	<form action="{$url_submit|escape:'html':'UTF-8'}" method="post" enctype="multipart/form-data" class="form-horizontal">
+	<form action="index.php/international/translations/extract" method="post" enctype="multipart/form-data" class="form-horizontal">
```

Comment:
I plan to make the Symfony router totaly available and to remove this ugly hack on ``Link`` class.
But because it's huge, risky and totaly unrelated to what I'm doing (allow the translations of themes in back office), for now hardcode this url is "good enough" :)